js/net_test.ts

```diff
@@ -193,24 +195,22 @@ testPerm({ net: true }, async function netCloseWriteSuccess() {
     err = e;
   }
   assert(!!err);
-  assertEquals(err.kind, Deno.ErrorKind.BrokenPipe);
-  assertEquals(err.name, "BrokenPipe");
+  assertEquals(err, new Error("write error"));
```

Comment:
Is it better to handle such error as DenoError?
https://github.com/denoland/deno/blob/master/js/files.ts#L163

Reply:
Yeah

Looks like you're failing on Windows. I forget what the deal with this test was. Maybe just flaky. Try running it couple of times locally.

Reply:
Ok

I found the cause of failing on Windows is same to #2358. 
I'll try to deal with it.